Funcky.Analyzers/Funcky.Analyzers/AnalyzerReleases.Unshipped.md

```diff
@@ -0,0 +1,10 @@
+; Unshipped analyzer release
+; https://github.com/dotnet/roslyn-analyzers/blob/main/src/Microsoft.CodeAnalysis.Analyzers/ReleaseTrackingAnalyzers.Help.md
+
+### New Rules
+
+Rule ID | Category | Severity | Notes
+--------|----------|----------|-------
+λ1001 | Funcky | Warning | EnumerableRepeatOnceAnalyzer
+λ1002 | Funcky | Warning | EnumerableRepeatNeverAnalyzer
+λ1003 | Funcky | Warning | UseWithArgumentNamesAnalyzer
```

Comment:
```suggestion
| Rule ID | Category | Severity | Notes                         |
|---------|----------|----------|-------------------------------|
| λ1001   | Funcky   | Warning  | EnumerableRepeatOnceAnalyzer  |
| λ1002   | Funcky   | Warning  | EnumerableRepeatNeverAnalyzer |
| λ1003   | Funcky   | Warning  | UseWithArgumentNamesAnalyzer  |
```

Reply:
Automatically generated, the change leads to the warning:
warning RS2007: Analyzer release file 'AnalyzerReleases.Unshipped.md' has a missing or invalid release header '| Rule ID | Category | Severity | Notes                         |